src/sad.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021, The rav1e contributors. All rights reserved
```

Comment:
We already have some sad functions in asm/dist/mod.rs. Some documentation on these functions about why this is separate (e.g. it's per line, doesn't operate on PlaneRegion) would be useful.

Reply:
I renamed the module to `sad_row`, and I updated the doc comment for the `sad_row` function